src/components/sbb-breadcrumb/readme.md

```diff
@@ -0,0 +1,80 @@
+The `sbb-breadcrumb` is a component used to display a link to a page; when it's used within the `sbb-breadcrumb-group` component, 
+it can display the list of the links the user visited to arrive at the current page.
+
+It is possible to provide a text via an unnamed slot; the component can optionally display a `<sbb-icon>`
+at the component start using the `iconName` property or via custom content using the `icon` slot.
+Text and icon are not exclusive and can be used together.
+
+It's possible to set all the link related properties (`download`, `href`, `rel` and `target`);
+the `ariaCurrent` property is used to make the breadcrumb read correctly by screen-readers when the component
+is used in the `sbb-breadcrumb-group`.
+
+## Usage
+
+Breadcrumb with text:
+
+```html
+<sbb-breadcrumb href='/contact' target='_blank'>
+  Contact us
+</sbb-breadcrumb>
+```
+
+Breadcrumb with icon:
+
+```html
+<sbb-breadcrumb href='/download' download icon-name='download-small'></sbb-breadcrumb>
```

Comment:
A little weird example as you will probably never have a download attribute in the breadrcrumb :)

Reply:
Fixed in eb3f825a